lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -139,29 +111,84 @@ def get_thread_created_event_data(thread, followed):
         # However, the view does not contain that data, and including it will
         # likely require changes elsewhere.
     }
+    track_created_event(request, event_name, course, thread, event_data)
 
 
-def get_comment_created_event_name(comment):
-    """Get the appropriate event name for creating a response/comment"""
-    return COMMENT_CREATED_EVENT_NAME if comment.get("parent_id") else RESPONSE_CREATED_EVENT_NAME
-
-
-def get_comment_created_event_data(comment, commentable_id, followed):
+def track_comment_created_event(request, course, comment, commentable_id, followed):
     """
-    Get the event data payload for comment creation (excluding fields populated
-    by track_forum_event)
+    Send analytics event for a newly created response or comment.
     """
+    obj_type = 'comment' if comment.get("parent_id") else 'response'
+    event_name = _EVENT_NAME_TEMPLATE.format(obj_type=obj_type, action_name='created')
     event_data = {
         'discussion': {'id': comment.thread_id},
         'commentable_id': commentable_id,
         'options': {'followed': followed},
     }
-
-    parent_id = comment.get("parent_id")
+    parent_id = comment.get('parent_id')
     if parent_id:
         event_data['response'] = {'id': parent_id}
+    track_created_event(request, event_name, course, comment, event_data)
+
+
+def track_voted_event(request, course, obj, vote_value, undo_vote=False):
+    """
+    Send analytics event for a vote on a thread or response.
+    """
+    if isinstance(obj, cc.Thread):
+        obj_type = 'thread'
+    else:
+        obj_type = 'response'
```

Comment:
Is there a reason why we only have `'response'` and not `'comment'` here as well?